lib/modules/manager/fleet/extract.ts

```diff
@@ -50,6 +52,22 @@ function extractFleetHelmBlock(doc: FleetHelmBlock): PackageDependency {
       skipReason: 'missing-depname',
     };
   }
+
+  if (isOCIRegistry(doc.chart)) {
+    const ociImageRegistry = doc.chart.replace('oci://', '');
+    const dockerDep = getDep(`${ociImageRegistry}:${doc.version}`, false);
+
+    return {
+      ...dockerDep,
+      depType: 'fleet',
```

Comment:
shouldn't it be helm chart?

Reply:
I wasn't sure tbh. The manager uses "fleet" for the other helm charts as well, see:

https://github.com/renovatebot/renovate/blob/5dfe40a6dc2a6ddb80e801cd09a7605fa38622f9/lib/modules/manager/fleet/extract.ts#L43

I thought that if I didn't know why the depType "fleet" was created in the first place, it would be a good idea to keep it at least consistent.

Reply:
Should be `fleet` as this is a reference to that this are fleet files.